sentinelhub/api/batch_statistical.py

```diff
@@ -0,0 +1,149 @@
+
+    def get_status(self, batch_request: BatchStatisticalRequestType) -> JsonDict:
+        """Collects information about a the status of a request
+
+        `Batch Statistical API reference
+        <https://docs.sentinel-hub.com/api/latest/reference/#operation/batchStatisticalGetStatus>`__
+
+        :return: Batch request status dictionary
+        """
+        request_id = self._parse_request_id(batch_request)
+        endpoint_url = f"{self._get_processing_url(request_id)}/status"
+        request_info = self.client.get_json_dict(url=endpoint_url, use_session=True)
+        return request_info
+
+    def start_analysis(self, batch_request: BatchStatisticalRequestType) -> Json:
+        """Starts analysis of a batch job request
+
+        `Batch Statistical API reference
+        <https://docs.sentinel-hub.com/api/latest/reference/#operation/batchStatisticalAnalyse>`__
+
+        :param batch_request: It could be a batch request object, a raw batch request payload or only a batch
+            request ID.
+        """
+        return self._call_job(batch_request, "analyse")
+
+    def start_job(self, batch_request: BatchStatisticalRequestType) -> Json:
+        """Starts running a batch job
+
+        `Batch Statistical API reference
+        <https://docs.sentinel-hub.com/api/latest/reference/#operation/batchStartStatisticalRequest>`__
+
+        :param batch_request: It could be a batch request object, a raw batch request payload or only a batch
+            request ID.
+        """
+        return self._call_job(batch_request, "start")
+
+    def cancel_job(self, batch_request: BatchStatisticalRequestType) -> Json:
+        """Cancels a batch job
+
+        `Batch Statistical API reference
+        <https://docs.sentinel-hub.com/api/latest/reference/#operation/batchCancelStatisticalRequest>`__
+
+        :param batch_request: It could be a batch request object, a raw batch request payload or only a batch
+            request ID.
+        """
+        return self._call_job(batch_request, "cancel")
+
+
+@dataclass_json(letter_case=LetterCase.CAMEL, undefined=Undefined.INCLUDE)
+@dataclass(repr=False)
+class BatchStatisticalRequest(BaseBatchRequest):  # pylint: disable=abstract-method
+    """A dataclass object that holds information about a batch request"""
+
+    request_id: str = field(metadata=dataclass_config(field_name="id"))
+    other_data: CatchAll
+    completion_percentage: float
+    request: dict
+    status: BatchRequestStatus = field(metadata=enum_config(BatchRequestStatus))
+    user_id: Optional[str] = None
+    created: Optional[dt.datetime] = field(metadata=datetime_config, default=None)
+    cost_pu: Optional[float] = None
+    user_action: Optional[BatchUserAction] = field(metadata=enum_config(BatchUserAction), default=None)
+    user_action_updated: Optional[str] = field(metadata=datetime_config, default=None)
+    error: Optional[str] = None
+
+    _REPR_PARAM_NAMES = (
```

Comment:
I see you are using a tuple here and it is better than a list in `BatchRequest` class. Would you mind changing to tuple there as well?

Reply:
done